cli/types/cypress.d.ts

```diff
@@ -2500,7 +2500,7 @@ declare namespace Cypress {
     action: 'select' | 'drag-drop'
   }
 
-  interface BlurOptions extends Loggable, Forceable { }
+  interface BlurOptions extends Loggable, Timeoutable, Forceable { }
```

Comment:
[`cy.blur`](https://docs.cypress.io/api/commands/blur#Arguments) supports `timeout` but it was missing in types. I added it.

Reply:
hmm, what does `.timeout` actually do in `cy.blur`? it gets passed to Cypress.log, but shouldn't this be a synchronous operation? maybe we need to remove this if it's a no-op.

Reply:
I also think it's a synchronous operation and harder to finish blur() over 4 seconds. 

But it might be here because we need it for `retry` and `verifyAssertions`. 

And it's also in the documentation. Removing it is a breaking change.

Reply:
IDK, I feel like if it's a no-op currently, removing the option isn't really breaking. But it's ok, don't need to worry about it here.